yappi/_yappi.c

```diff
@@ -1752,6 +1870,40 @@ set_context_name_callback(PyObject *self, PyObject *args)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+set_context_backend(PyObject *self, PyObject *args)
+{
+    _subsystem_type_t input_sub_type;
+    
+    if (!PyArg_ParseTuple(args, "i", &input_sub_type)) {
+        return NULL;
+    }
+    
+    if (input_sub_type == subsystem_type)
+    {
+        Py_RETURN_NONE;
+    }
+    
+    // TODO(suhail.muhammed): Is this the correct condition to check?

```

Comment:
Yes. But I think we also need to add a check for: there should be no stats available. all stats cleared. See: `yapphavestats`

Reply:
I'll add this as well.